src/cirrus/plugins/deployers/chef.py

```diff
@@ -28,6 +28,16 @@
 LOGGER = get_logger()
 
 
+def attr_list(x):
+    """
+    parser for attribute lists on cli
+    """
```

Comment:
Trying to avoid empty splits like this?
```
In [7]: 'x, y, , x'.split(',')
Out[7]: ['x', ' y', ' ', ' x']
```

I think you can get away with removing the len() `if y.strip()`